DeviceSDK/src/main/java/org/astarteplatform/devicesdk/transport/mqtt/AstarteMqttTransport.java

```diff
@@ -47,9 +49,9 @@ public void onFailure(IMqttToken asyncActionToken, Throwable exception) {
           if (listener != null) {
             listener.onFailure(derivedException);
           } else {
-            derivedException.printStackTrace();
+            logger.severe("Error during transport failure: " + derivedException.getMessage());
           }
-          exception.printStackTrace();
+          logger.severe("Original exception: " + exception.getMessage());
```

Comment:
Change Original with Starting
```suggestion
          logger.severe("Starting exception: " + exception.getMessage());
```

Reply:
Change the log message to "Starting.."